Rocket.Chat/External/RCEmojiKit/Views/EmojiPicker/EmojiCollectionViewCell.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  EmojiCollectionViewCell.swift
+//  Rocket.Chat
+//
+//  Created by Samar Sunkaria on 3/24/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+import FLAnimatedImage
+
+class EmojiCollectionViewCell: UICollectionViewCell {
+    enum Emoji {
+        case custom(URL?)
+        case standard(String?)
+    }
+
+    var emojiLabel: UILabel = {
+        let lbl = UILabel()
+        lbl.lineBreakMode = .byClipping
+        lbl.textAlignment = .center
+        lbl.baselineAdjustment = .alignCenters
+        lbl.font = UIFont.systemFont(ofSize: 32)
+        lbl.backgroundColor = UIColor.white
+        return lbl
```

Comment:
Can you change from `lbl` to `emojiLabel`, please?

Reply:
I'll change it.

though, I've seen a general pattern in swift dev for using a small variable names in a closure initialization (eg v for a view), mostly because the context is very clear and it reduces the horizontal space required.